### PR TITLE
✨ Introduce Security hub

### PR DESCRIPTION
**Pull request by jgalat**
This PR introduces the security hub
Including
- Hub with information from all our contracts and current auditing state
- New section for reviewing and revoking unwanted allowances on our contracts
- Protocol activity monitor for reviewing changes applied (and queued to be) to the protocol
- Introduce a news modals for new announcements

### CODE REVIEW

File: components/Navbar/MoreMenu/index.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    (event: React.MouseEvent<HTMLButtonElement>) => {
+      if (anchorEl !== event.currentTarget) {
+        setAnchorEl(event.currentTarget);
+      }
+    },
+    [anchorEl],
+  );
+  const closeMenu = useCallback(() => setAnchorEl(null), [setAnchorEl]);
+
+  const handleOnHover = useCallback(() => clearTimeout(currTimeout), [currTimeout]);
```
Comment (sebipap) [inline components/Navbar/MoreMenu/index.tsx:37]:
```suggestion
  const handleHover = useCallback(() => clearTimeout(currTimeout), [currTimeout]);
```

Comment (sebipap) [reply components/Navbar/MoreMenu/index.tsx:37]: same for the other one, this is for consistency but not very important, skip if needed

**Review by franm91 — APPROVED** [review]

File: components/NewsModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+const NewsModal = () => {
+  const { t } = useTranslation();
+  const news = useMemo(
+    () => [
+      {
+        id: 'Security Hub',
+        title: t('Security Hub'),
+        description: [
+          t('Stay up to date with all the security measures we put in place to keep the Protocol safe.'),
+          t("From smart contract audits to revoking token allowances, you'll find all the details right here."),
```
Comment (sebipap) [inline components/NewsModal/index.tsx:30]: why did this end up with double quotes? weird prettier didn't fix that 🤔

File: components/NewsModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+  const { breakpoints } = useTheme();
+  const isMobile = useMediaQuery(breakpoints.down('md'));
+
+  const markNewsAsRead = useCallback((titles: string[]) => {
+    const readNews = getReadNews();
+    const newReadNews = [...new Set([...readNews, ...titles])];
+    localStorage.setItem(NEWS_READ_KEY, JSON.stringify(newReadNews));
+  }, []);
+
+  const filteredNews = news.filter(({ until, title }) => new Date(until) > new Date() && !isNewsRead(title));
```
Comment (sebipap) [inline components/NewsModal/index.tsx:77]:
```suggestion
  const unreadNews = news.filter(({ until, title }) => new Date(until) > new Date() && !isNewsRead(title));
```

File: components/NewsModal/index.tsx [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+  const [readIds, setReadIds] = useState<string[]>(filteredNews[0]?.id ? [filteredNews[0].id] : []);
+
+  const closeModal = useCallback(() => {
+    markNewsAsRead(readIds);
+    setOpen(false);
+  }, [markNewsAsRead, readIds]);
+
+  const selectedNews = useMemo(() => filteredNews[selected], [filteredNews, selected]);
+
+  const handleSetSelected = useCallback(
```
Comment (sebipap) [inline components/NewsModal/index.tsx:87]:
```suggestion
  const handleIndexChange = useCallback(
```

File: components/RiskFeed/api/index.tsx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+};
+
+type Parameter = {
+  name: string;
+  type: string;
+  value: string;
+  valueDecoded?: ValueDecoded[];
+};
+
+type ValueDecoded = {
```
Comment (sebipap) [inline components/RiskFeed/api/index.tsx:103]:
```suggestion
type DecodedValue = {
```

File: components/RiskFeed/api/index.tsx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+type TxData = {
+  hexData: Hex;
+  dataDecoded: DataDecoded | null;
+  to: Wallet;
+  value: string;
+  operation: number;
+  addressInfoIndex: Record<Address, Wallet>;
+};
+
+export type DataDecoded = {
```
Comment (sebipap) [inline components/RiskFeed/api/index.tsx:91]:
```suggestion
export type DecodedData = {
```

**Review by sebipap — APPROVED** [review]
LGTM ✅

added non-blocking comments

File: components/NewsModal/index.tsx (hunk 2 above)
Comment (jgalat) [reply components/NewsModal/index.tsx:30]: there is a single quote in `you'll`

Comment (sebipap) [reply components/NewsModal/index.tsx:30]: ohh, ok!

**Pull request merged**